tests/unit/test_pull_request.py

```diff
@@ -30,13 +31,37 @@ def test_repository_client__init__(repository: Repo, mock_github_repo: Repositor
     act: when RepositoryClient is initialized
     assert: RepositoryClient is created and git user is configured.
     """
+    pull_request.RepositoryClient(repository=repository, github_repository=mock_github_repo)
+
+    config_reader = repository.config_reader()
+    assert (
+        config_reader.get_value(*pull_request.CONFIG_USER_NAME) == pull_request.ACTIONS_USER_NAME
+    )
+    assert (
+        config_reader.get_value(*pull_request.CONFIG_USER_EMAIL) == pull_request.ACTIONS_USER_EMAIL
+    )
+
+
+def test_repository_client__init__name_email_set(repository: Repo, mock_github_repo: Repository):
+    """
+    arrange: given a local git repository client with the user and email configuration already set
+        and mock github repository client
+    act: when RepositoryClient is initialized
+    assert: RepositoryClient is created and git user is not reconfigured.
```

Comment:
The phrasing `git user is not reconfigured` is kind of strange
Perhaps `git user configuration is kept` ?

Reply:
How about `... git user configuration is not overridden`?

Reply:
Good :)